dockerfile-latest

```diff
@@ -0,0 +1,41 @@
+FROM airsharedcontainers.azurecr.io/vstsdropdownloadcore:master AS app
+FROM airsharedcontainers.azurecr.io/msftkube AS msftkube
```

Comment:
So confused how this dockerfile relates to the template. Is this replacing the other or is it complementary.
Where is the build def that produces airsharedcontainers.azurecr.io/msftkube. 
Aren't we losing the versioing of msftkube (do we care?)

Reply:
Complimentary

Reply:
We don't lose the versioning because that is handled inside of our VSTS task. We will still tag based on the msftkube version (Build.TriggeredBy.BuildId is the msftkube build version)